howto/anbox/harden.md

```diff
@@ -0,0 +1,31 @@
+(howto-harden)=
+# How to harden your deployment
+
+Anbox Cloud is secure by design, however, there are certain aspects of the deployment that you need to harden to ensure security.
+
+## Reduce attack surface
+
+The following diagram illustrates the attack surface and potential points of threat.
+
+![Anbox Cloud attack surface|690x398](/images/anbox_attack_surface.svg)
+
+The threat points are:
+
+### 1 - Web client
```

Comment:
There is nothing in particular which comes to my mind here.

However we should mention the OS level and maybe reference other parts of the Ubuntu Pro offering, e.g. livepatch/usg/cis (https://documentation.ubuntu.com/pro/pro-client/enable_cis/)

Reply:
We can maybe mention here that gateway API tokens should never used in the client and communication with the gateway / authentication need to come through an intermediate service, e.g. see https://documentation.ubuntu.com/anbox-cloud/tutorial/stream-client/